docs/content/preview/admin/yb-admin.md

```diff
@@ -1325,52 +1326,55 @@ Changes the universe replication for the specified source universe. Use this com
 * Add or remove tables in an existing replication UUID
 * Modify the source master addresses
 
-If no tables have been configured for replication, use [`setup_universe_replication`](#setup-universe-replication).
+If no tables have been configured for replication, use [setup_universe_replication](#setup-universe-replication).
 
-To check if any tables are configured for replication, use [`list_cdc_streams`](#list-cdc-streams).
+To check if any tables are configured for replication, use [list_cdc_streams](#list-cdc-streams).
 
 **Syntax**
 
 Use the `set_master_addresses` subcommand to replace the source master address list. Use this if the set of masters on the source changes:
 
 ```sh
-yb-admin -master_addresses <master-addresses> \
-    alter_universe_replication <source_universe_uuid> \
+yb-admin -master_addresses <target_master_addresses> \
+    alter_universe_replication <source_universe_uuid>_<replication_name> \
     set_master_addresses <source_master_addresses>
 ```
 
-* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
 * *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name of the replication to be altered.
 * *source_master_addresses*: Comma-separated list of the source master addresses.
 
 Use the `add_table` subcommand to add one or more tables to the existing list:
 
 ```sh
-yb-admin -master_addresses <master-addresses> \
-    alter_universe_replication <source_universe_uuid> \
-    add_table [comma_separated_list_of_table_ids] \
-    [comma_separated_list_of_producer_bootstrap_ids]
+yb-admin -master_addresses <target_master_addresses> \
+    alter_universe_replication <source_universe_uuid>_<replication_name> \
+    add_table [ <comma_separated_list_of_table_ids> ] \
+    [ <comma_separated_list_of_producer_bootstrap_ids> ]
 ```
 
-* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
 * *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name of the replication to be altered.
 * *comma_separated_list_of_table_ids*: Comma-separated list of table identifiers (`table_id`).
-* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of bootstrap identifiers (`bootstrap_id`). These are obtained from using [`bootstrap_cdc_producer`](#bootstrap-cdc-producer).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of bootstrap identifiers (`bootstrap_id`). Obtain these using [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids).
```

Comment:
I think it would be good to mention that use the source universe master addresses for bootstrap_cdc_producer, so the user does not accidentally use target universe master addresses to get the bootstrap ids.

Reply:
Updated to specify source universe